### ISSUE TITLE
Distributed application is somewhat slow

### ISSUE THREAD

**Issue by mpetrunic** [NodeFactoryIo/ChainGuardian#231]
It takes long time to register and login (4-5s). We should display loading in those situations.
We should also investigate why is so slow, maybe native crypto is not used.

**Comment by morrigan** [comment]
Which part is slow? Mnemonic generation screens or?

**Comment by mpetrunic** [comment]
When you confirm second password you are just staying there for 4-5s, then you click login and you wait anothe 3s same goes for showing private key, rest of stuff is blazing fast. Seems like only crypto related stuff is slow

**Comment by morrigan** [comment]
In a bundled app? Because works fine in development

**Comment by mpetrunic** [comment]
> In a bundled app? Because works fine in development

Probably because in development, it uses native dependencies while for some reason when bundled it falls back to js. I think we should display loading none the less and investigate performance. Might be there is no native build for my version of ubuntu (we will see when we try mac or windows version)

**Comment by morrigan** [comment]
Yeah, possibly the same thing what happened with React native and ethers :D

**Comment by morrigan** [comment]
Is this duplicate of #241 ?

**Comment by mpetrunic** [comment]
#241 just mitigates this we should investigate why is slow later on
